dream-web/dune

```diff
@@ -1,3 +1,9 @@
+(rule
+ (target static.ml)
+ (deps (source_tree static))
+ (action (with-stdout-to %{null}
+  (run ocaml-crunch --mode=plain --output=%{target} static))))
```

Comment:
You don't need the `with-stdout-to`, and you can add the `--silent` flag to `ocaml-crunch`.